learn/tanssi/technical-features.md

```diff
@@ -70,7 +70,7 @@ Among many other responsibilities, the relay chain validates and finalizes the b
 
 ![Relay chain](/images/learn/tanssi/technical/technical-4.webp)
 
-As mentioned, the Tanssi network and the appchains are sibling chains with no hierarchical dependency. They are communicated via the relay chain, and therefore, the relay chain plays a key role in the protocol.
+As mentioned, the Tanssi network and the appchains are sibling chains with no hierarchical dependency. They communicate via the relay chain, and therefore, the relay chain plays a key role in the protocol.
```

Comment:
Thanks - @augustocollado please include this typo in one of the PRs we have open

Reply:
done!